bindings/nodejs/types/participation.ts

```diff
@@ -0,0 +1,52 @@
+import type { OutputId } from './output'
+
+export interface ParticipationOverview {
+    participations: [EventId, [OutputId, ParticipationOverview]]
+}
+
+export interface TrackedParticipationOverview {
+    blockId: string
+    amount: string
+    startMilestoneIndex: number
+    endMilestoneIndex: number
+}
+
+export interface Event {
+    id: EventId
+    data: EventData
+}
+
+export type EventId = string
+
+export interface EventStatus {
+    milestoneIndex: number
+    status: string
+    questions?: Answer[]
+    checksum: string
+}
+
+interface EventData {
+    name: string
+    milestoneIndexCommence: number
+    milestoneIndexStart: number
+    milestoneIndexEnd: number
+    payload: VotingEventPayload
```

Comment:
This could also be a staking payload, maybe you can copy the types from the mainnet Firefly version?

Reply:
But do we want it to be a staking payload if we don't provide support for it? I don't think so, which is why at the moment, it's better to type it as `VotingEventPayload`. Once we want a staking payload, this should be updated.

Reply:
But it can return staking payloads already, can't this break something for you if that's happening?

Reply:
If we don't want staking events, how can it return a staking payload? Basically we are introducing extra code that we don't want to use anyway (for the foreseeable future). I suggest that we remove this from iota.rs (happy to do it myself as well): https://github.com/iotaledger/iota.rs/blob/91bd73383a95ee9de4bfba90c1bc9b922e8d4ca3/client/src/node_api/participation/types/mod.rs#L56. Maybe I missed a reason why we need to keep it, so enlighten me in that case pls :D

Reply:
Someone can create a staking event on the participation plugin and then you get these from the node and it would return an error, because you only expect voting events